### DISCUSSION TITLE
Can this repository be used for be submitting feature requests?

### DISCUSSION THREAD

**Discussion by just1a-person** [microsoft/Windows-Dev-Performance#71] [Q&A] [answered]
I know about Feedback Hub. But GitHub feels more natural to me

**Comment by zooba** [comment, marked as answer]
We're trying to keep the scope of this repository fairly controlled, because we only have a very small group of people looking at it. If/when it grows as big as Feedback Hub, we'll have to pass off review to other people, which will hurt the experience for everyone.

Rich is in charge though, so if he says go for it, then go for it. Until then, I'd say that if you can't phrase it as "developing on Windows is really hard because XYZ doesn't work", hang onto it for now (or put it into Feedback Hub and try to promote it for votes).
